templates/web-application/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "{{PACKAGE-NAME}}",
+  "version": "0.0.0",
+  "description": "",
+  "type": "module",
+  "scripts": {
+    "dev": "webpack serve --config webpack.dev.js",
+    "build": "webpack --config webpack.build.js",
+    "test": "jest",
+    "serve-build": "pnpm http-server dist -p 8080 -P http://localhost:8080?",
+    "reset": "pnpm dlx rimraf node_modules dist",
+    "lint": "eslint .",
+    "analyze": "webpack --config webpack.build.js --analyze",
+    "postinstall": "pnpm dlx rimraf node_modules/.cache"
+  },
+  "dependencies": {
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-router-dom": "6.11.0"
+  },
+  "devDependencies": {
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
+    "@swc/core": "^1.3.56",
+    "@swc/jest": "^0.2.26",
+    "@swc/helpers": "^0.5.1",
```

Comment:
`@swc/helpers` is required to fix error on SWC side (`Can't resolve '@swc/helpers/_/_sliced_to_array'`) at the build time.

Without this, this `const [response, setResponse] = useState("We don't have the response");
` (from `src/components/TheAnswer.tsx`) will crash at the build time.

Reply:
I don't know, but I still get this error continuously even after resetting the project (`pnpm reset`) or trying on a different computer.